client/gutenberg/extensions/markdown/jetpack-markdown-block.jsx

```diff
@@ -9,10 +9,11 @@ import { registerBlockType } from '@wordpress/blocks';
 /**
  * Internal dependencies
  */
+import './markdown-editor.scss';
 import JetpackMarkdownBlockEditor from './jetpack-markdown-block-editor';
 import JetpackMarkdownBlockSave from './jetpack-markdown-block-save';
 
-registerBlockType( 'jetpack/markdown-block', {
+registerBlockType( 'a8c/markdown', {
```

Comment:
Do we also want to apply that to any other extensions? Publicize still uses `jetpack-publicize`, and we might want to be consistent with that.

Reply:
Good catch! Updated.

Reply:
Seems like this introduced a bug 😕 

See #26726